channeldb/payments.go

```diff
@@ -536,6 +536,14 @@ type PaymentsQuery struct {
 	// CountTotal indicates that all payments currently present in the
 	// payment index (complete and incomplete) should be counted.
 	CountTotal bool
+
+	// MinCreationTimeNs, if set, filters out all payments with a
+	// creation time less than it.
+	MinCreationTimeNs int64
```

Comment:
Similair comments to the invoice filtering PR, use `time.Time` here to begin with rather than needing to convert it below.

Reply:
Done.